ercc/attestation/verifier.go

```diff
@@ -37,7 +37,7 @@ func (v *VerifierImpl) VerifyEvidence(evidenceBytes, expectedStatementBytes []by
 
 	expectedMrEnclaveBytes := []byte(expectedMrEnclave)
 	expectedMrEnclavePtr := C.CBytes(expectedMrEnclaveBytes)
-	defer C.free(expectedStatementPtr)
+	defer C.free(expectedMrEnclavePtr)
 	expectedMrEnclaveLen := len(expectedMrEnclaveBytes)
```

Comment:
good catch! #474 contains the fix already.

Reply:
The "update branch" automatically solved the conflict.